svg/elements/path.json

```diff
@@ -0,0 +1,101 @@
+{
+  "svg": {
+    "elements": {
+      "path": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Element/path",
+          "support": {
+            "webview_android": {
+              "version_added": "3"
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1.5"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": "8"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "3.1"
+            },
+            "safari_ios": {
+              "version_added": "3.1"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
```

Comment:
The `pathLength` attribute is missing.

SVG 1.1 also defined `externalResourcesRequired` and `transform` as attributes, which got removed in SVG 2. I'm not sure whether any browsers ever supported the former (or even still support them), but the latter is definitely still supported in browsers, see the examples at https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform. So, I guess we should include them.

Reply:
I haven't made up my mind about those two attributes. I'm considering provide compat information about `externalResourcesRequired` into the global attribute part as it is shared among many elements. For transform, I'm still wondering how to handle presentation attributes that can be used as CSS properties.

At that point I prefer not to add those two attributes and adding them back if it appears to be necessary.